kukkuu/settings.py

```diff
@@ -92,6 +96,13 @@
 ILMOITIN_TRANSLATED_FROM_EMAIL = env("ILMOITIN_TRANSLATED_FROM_EMAIL")
 ILMOITIN_QUEUE_NOTIFICATIONS = env("ILMOITIN_QUEUE_NOTIFICATIONS")
 
+if env.str("DEFAULT_SMS_SENDER"):
```

Comment:
Just a minor detail, but since `DEFAULT_SMS_SENDER` is already defined as string, this can be just `if env("DEFAULT_SMS_SENDER"):`